string groupName
string droneName
string ip
string droneType
---
string result
